SearchableToolsUITests/SearchableToolsUITests.swift

```diff
@@ -8,34 +8,16 @@
 import XCTest
 
 final class SearchableToolsUITests: XCTestCase {
-
-    override func setUpWithError() throws {
-        // Put setup code here. This method is called before the invocation of each test method in the class.
-
-        // In UI tests it is usually best to stop immediately when a failure occurs.
-        continueAfterFailure = false
-
-        // In UI tests it’s important to set the initial state - such as interface orientation - required for your tests before they run. The setUp method is a good place to do this.
-    }
-
-    override func tearDownWithError() throws {
-        // Put teardown code here. This method is called after the invocation of each test method in the class.
-    }
-
-    func testExample() throws {
-        // UI tests must launch the application that they test.
+    
+    func test_searchQuery_ToolName () {
```

Comment:
Let's rename this function name
```suggestion
    func test_searchForMartilloTool_displaysTool () {
```